src/main/scala/beam/utils/scenario/UrbanSimScenarioLoader.scala

```diff
@@ -161,13 +165,10 @@ class UrbanSimScenarioLoader(
           )
           .toBuffer
 
-        beamScenario.vehicleTypes.values
-          .find(_.vehicleCategory == VehicleCategory.Bike) match {
-          case Some(vehType) =>
-            vehicleTypes.append(vehType)
-          case None =>
-            throw new RuntimeException("Bike not found in vehicle types.")
+        if (rand.nextDouble() <= beamScenario.beamConfig.beam.agentsim.agents.vehicles.bicycleHavingProbability) {
```

Comment:
add simple test: make bike attractive (increase bike intercept) and compare scenario with 0% bike vs. 100%

Reply:
Here are the results, fraction equals to 100% (1.0)
![image](https://user-images.githubusercontent.com/10290975/87857848-ec2c0d80-c953-11ea-982c-b18fb8b5ba20.png)
Fraction equals to 0.0:
![image](https://user-images.githubusercontent.com/10290975/87857853-0239ce00-c954-11ea-94b6-51aec3564f7f.png)
Fraction equals to 0.5:
![image](https://user-images.githubusercontent.com/10290975/87857885-5349c200-c954-11ea-870c-c97721d59f88.png)